src/sonic-yang-models/yang-models/sonic-port.yang

```diff
@@ -91,6 +91,14 @@ module sonic-port{
 					}
 				}
 
+				leaf dhcp_rate_limit {
+					description "DHCP DOS Mitigation Rate with default value 300";
+					type uint32 {
+						range 0..800000;
```

Comment:
Why do you use 800000 as max value?

Reply:
There is no specific max standard range for DHCP DOS Range. We added a reasonably large maximum range vale to cater all device rate limit scenarios.

Reply:
It seems that we don't really need this range.

Reply:
Yes it can work without range as well. If suggested, we can remove range.